ui/lib/config-ui/addon/components/messages/page/details.hbs

```diff
@@ -0,0 +1,49 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+<Messages::TabPageHeader
+  @authenticated={{@message.authenticated}}
+  @pageTitle={{@message.title}}
+  @breadcrumbs={{@breadcrumbs}}
+/>
+
+<Toolbar>
+  <ToolbarActions aria-label="message delete and edit">
+    {{#if @message.canDeleteCustomMessages}}
+      <ConfirmAction
+        class="toolbar-button"
+        @buttonColor="secondary"
+        @onConfirmAction={{perform this.deleteMessage}}
+        @confirmTitle="Are you sure?"
+        @confirmMessage="This will delete this message permanently. You cannot undo this action."
+        @buttonText="Delete message"
+        data-test-confirm-action="Delete message"
+      />
+      <div class="toolbar-separator"></div>
+    {{/if}}
+    {{#if @message.canEditCustomMessages}}
+      <LinkTo class="toolbar-link" @route="messages.message.edit" @model={{@message.id}} data-test-link="edit">
+        Edit message
+        <Icon @name="chevron-right" />
+      </LinkTo>
+    {{/if}}
+  </ToolbarActions>
+</Toolbar>
+
+{{#each @message.allFields as |attr|}}
+  {{#if (or (eq attr.name "endTime") (eq attr.name "startTime"))}}
+    <InfoTableRow
+      @label={{capitalize (humanize (dasherize attr.name))}}
+      @value={{if
+        (and (eq attr.name "endTime") (not (get @message attr.name)))
```

Comment:
This is a little hard to read. Maybe add a comment to help someone later down the road understand what is displayed here?